### PR TITLE
refactor: 💡 Update manage dropdown for host-catalogs

### PR DESCRIPTION
**Pull request by CaptFalc**
✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14664

# Description
Updates Manage dropdown in Host-Catalogs to HDS

## Screenshots (if appropriate)
**Before**
host catalog before1
host catalog before2

**After**
host catalog after1
host catalog after2

## How to Test
1. Start Boundary Client
2. Select org from sidebar and select an org in table.
3. Select project from sidebar and select an project in table.
4. Select Host catalog from sidebar and select a host catalog
5. HDS Dropdown should be reflected 

## Checklist

- [X] I have added before and after screenshots for UI changes
~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~

### CODE REVIEW

File: ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js [hunk 1]
```diff
@@ -94,6 +98,6 @@ module('Acceptance | host-catalogs | hosts | list', function (hooks) {
     instances.hostCatalog.authorized_collection_actions.hosts = ['create'];
     await visit(urls.hostCatalog);
     assert.ok(find(`[href="${urls.hosts}"]`));
-    assert.ok(find(`.rose-layout-page-actions [href="${urls.newHost}"]`));
+    assert.dom(MANAGE_DROPDOWN_SELECTOR).exists();
```
Comment (cameronperera) [inline ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js:101]: I think this test is verifying that we will still render the "New Host" action in the dropdown with only the `create` permission for hosts. We likely need to click the manage dropdown to open it, then do something like `assert.ok(find('[href="${urls.newHost}"]')`. The newer way of checking for the href that won't have a false positive is `assert.dom(YOUR_SELECTOR).hasAttribute('href', urls.newHost)`

**Review by cameronperera — CHANGES_REQUESTED** [review]
Looks great. Just a small change request for one of the test.

File: ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js (hunk 1 above)
Comment (CaptFalc) [reply ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js]: Thanks for the check Cameron!

File: ui/admin/tests/acceptance/host-catalogs/delete-test.js [hunk 2]
```diff
@@ -84,9 +89,7 @@ module('Acceptance | host-catalogs | delete', function (hooks) {
 
     await click(`[href="${urls.hostCatalog}"]`);
```
Comment (lisbet-alvarez) [inline ui/admin/tests/acceptance/host-catalogs/delete-test.js:91]: u also need to to add `await click(MANAGE_DROPDOWN_SELECTOR)` before asserting that the delete action does not exist.

Comment (CaptFalc) [reply ui/admin/tests/acceptance/host-catalogs/delete-test.js:91]: Completely missed that! Thank u lis🤗

**Review by lisbet-alvarez — APPROVED** [review]
Looks great!

**Review by cameronperera — APPROVED** [review]

**Pull request merged**